src/ert/simulator/simulation_context.py

```diff
@@ -155,28 +152,28 @@ def __len__(self) -> int:
 
     def isRunning(self) -> bool:
         # TODO: Should separate between running jobs and having loaded all data
-        return self._sim_thread.is_alive() or self._queue_manager.isRunning()
+        return self._sim_thread.is_alive() or self._job_queue.is_active()
 
     def getNumPending(self) -> int:
-        return self._queue_manager.getNumPending()
+        return self._job_queue.count_status(JobStatus.PENDING)  # type: ignore
```

Comment:
What is the reason for all the `type: ignore`? Will this be fixed in a separate PR/issue?

Reply:
These are due to difficulties with integrating the typing with the C-code and the custom `cwrap` (not Pybind). It will go away as soon as we have thrown away the C-code..

Reply:
Okay, great!

Reply:
Nope, we need it to retain the possibility to set it when running from _builder/_legacy.py